### ISSUE TITLE
beeping when the file is loaded

### ISSUE THREAD

**Issue by yao-eastside** [jeanyvesb9/Jupyter-Beeper#1]
In jupyter lab (have not tested on jupyter notebook yet), if I load a file that has a cell that imported this module (jupyter_beeper), it will make a beep automatically. It does not if I clean the cell's output and save. It will if I run the cell, save, and re-load.

What I expected was it should not make a beep sound when I load the file; it should only beep when I run the cell that make the beep() call.

**Comment by yao-eastside** [comment]
$ jupyter --version
jupyter core     : 4.6.1
jupyter-notebook : 6.0.2
qtconsole        : 4.5.5
ipython          : 7.8.0
ipykernel        : 5.1.2
jupyter client   : 5.3.4
jupyter lab      : 1.2.6
nbconvert        : 5.6.0
ipywidgets       : 7.5.1
nbformat         : 4.4.0
traitlets        : 4.3.3

**Comment by yao-eastside** [comment]
I am running on a Mac with latest OS. More sure about other OSes.

**Comment by jeanyvesb9** [comment]
Yes, indeed it was a known issue back when I developed the library. I was in a rush as I needed it working ASAP for an experimental physics lab setup, and I didn’t get to fix it in time.
I will take a look at it tomorrow. I have an idea that might work as a quick but proper fix.
